disperser/dataapi/subgraph_client.go

```diff
@@ -371,7 +390,7 @@ func convertBatches(subgraphBatches []*subgraph.Batches) ([]*Batch, error) {
 			Id:              []byte(batch.Id),
 			BatchId:         batchId,
 			BatchHeaderHash: []byte(batch.BatchHeaderHash),
-			BlockTimestamp:  ConvertNanosecondToSecond(timestamp),
```

Comment:
No conversion needed assuming timestamp from subgraph is in second (not nanosecond).